plugin-api/src/main/java/org/sonar/api/server/ws/WebService.java

```diff
@@ -312,6 +313,16 @@ public NewAction setHandler(RequestHandler h) {
       return this;
     }
 
+    /**
+     * Sets content type of the response. This is optional to do and the list of content types doesn't necessarily need to correspond to
```

Comment:
A bit odd to talk about a list while a single value is set. Maybe part of this javadoc should be on the enum instead.

Reply:
Good point